examples/providers/Cargo.toml

```diff
@@ -13,15 +13,21 @@ repository.workspace = true
 [dev-dependencies]
 alloy.workspace = true
 # Temp dependency fix to enable relevant features - Ref: https://github.com/alloy-rs/examples/pull/3#discussion_r1537842062
-alloy-rpc-client = { git = "https://github.com/alloy-rs/alloy", rev = "f7333c4", features = [
+alloy-rpc-client = { git = "https://github.com/alloy-rs/alloy", rev = "68952c0", features = [
     "pubsub",
+    "ipc",
+    "ws",
 ] }
-alloy-provider = { git = "https://github.com/alloy-rs/alloy", rev = "f7333c4", features = [
+alloy-provider = { git = "https://github.com/alloy-rs/alloy", rev = "68952c0", features = [
     "pubsub",
     "ws",
+    "ipc",
 ] }
 
+alloy-network = { git = "https://github.com/alloy-rs/alloy", rev = "68952c0" }
 
+alloy-transport = { git = "https://github.com/alloy-rs/alloy", rev = "68952c0" }
+alloy-transport-ipc = { git = "https://github.com/alloy-rs/alloy", rev = "68952c0" }
```

Comment:
Why are these required and not accessible on the `alloy` namespace?

Reply:
Compiler doesn't the recognize them as the same type when used with `alloy-provider` (w/o the namespace).

Reply:
Ah I see, then this should be resolved once that upstream PR has been merged